src/views/template/customer/customerorder.html

```diff
@@ -1,11 +1,16 @@
-<div class="panel panel-default">
-    <div class="panel-heading">
+<li class="list-group-item">
+    <div class="">
         {{ #customer }}
-            {{ name }}
+            <b>{{ name }}</b>
         {{ /customer }}
-        <span class="label label-warning pull-right">{{#l}}Earn{{/l}} {{#order}} {{ orderCost }} {{/order}}</span>
+        <button type="button" class="customer-serve btn btn-default btn-xs" style="float: right;" data-customer="{{ id }}">
+            <span class="text-success">
+                <span class="glyphicon glyphicon-ok"></span>
+                {{#l}}Earn{{/l}} {{#order}} {{ orderCost }} {{/order}}
+            </span>
+        </button>
     </div>
-    <div class="panel-body">
+    <div class="">
```

Comment:
I don't think the class attribute is needed here.

(or is it, and is it updated from JS?)